src/java/org/apache/cassandra/cql3/functions/FunctionParameter.java

```diff
@@ -168,7 +171,7 @@ static FunctionParameter anyType(boolean inferFromReceiver)
             public AbstractType<?> inferType(String keyspace,
                                              AssignmentTestable arg,
                                              @Nullable AbstractType<?> receiverType,
-                                             List<AbstractType<?>> previousTypes)
+                                             List<AbstractType<?>> inferredTypes)
```

Comment:
do we need a  annotation  of nullable for infferredType as line 60 ?

Reply:
We do, added